### DISCUSSION TITLE
这个网站默认是最快的吗？

### DISCUSSION THREAD

**Discussion by icls1337** [XIU2/CloudflareSpeedTest#342] [讨论]
speed.cloudflare.com

**Comment by XIU2** [comment]
你指的是什么？

**Comment by XIU2** [comment]
哪里？
如果你指的是这个域名解析的那两个 IP 就算了，你发这个 Issues 后我就试过了，延迟比自选的高 100ms+。。。

**Comment by icls1337** [comment]
> 哪里？

https://speed.cloudflare.com/

**Comment by XIU2** [comment]
我知道这个网站，我指的是你说的最快 IP 显示在哪里？
我没有看到网页上显示的有 Cloudflare CDN IP 呀。

另外，你单位说错了吧，300MB/s = 2.4Gbps 呢。。。

**Comment by XIU2** [comment]
没在 URL 中见到 IP 呀，该链接指向的依然是该网站域名的**两个解析 IP 之一**，你指的是这个 IP 吗？
> 这个域名的解析 IP 是固定的，因此这里只会显示 `104.16.148.27、104.16.149.27` 这两个 IP 之一。

![01](https://user-images.githubusercontent.com/54703944/143538147-52fac263-12bf-4c9c-9aec-f4c9e4fe530a.png)

测速结果也不出意外的很捉急。。。
![02](https://user-images.githubusercontent.com/54703944/143538233-07106387-70ee-469e-bf2c-785dcad85799.png)

**Comment by XIU2** [comment]
这是我随便自选了一个 IP 与其对比的：
```
# XIU2/CloudflareSpeedTest v2.0.1

开始延迟测速（模式：TCP IPv4，端口：443，平均延迟上限：9999 ms，平均延迟下限：0 ms)
3 / 3 [-------------------------------------------------------------------------------------------------------] 100.00%
开始下载测速（下载速度下限：0.00 MB/s，下载测速数量：10，下载测速队列：3）：
3 / 10 [------------------------------>________________________________________________________________________] 30.00%
IP 地址           已发送  已接收  丢包率  平均延迟  下载速度 (MB/s)
162.159.204.98    4       4       0.00    162.75    18.03
104.16.148.27     4       4       0.00    253.30    5.64
104.16.149.27     4       4       0.00    224.67    1.35
按下 回车键 或 Ctrl+C 退出。
```

这是指定自选 IP 后再去该网页测速的结果：
![03](https://user-images.githubusercontent.com/54703944/143538886-65e1608b-90ef-426a-91ba-4ea89549caaa.png)

**Comment by XIU2** [comment]
这个网站我很早就知道了，因为本项目一开始默认的**下载测速链接**就是这个网站的（上面截图里的那个 URL）。
不过后来发现，这网站竟然还限制每天测速总量，可能是用的人多了，一到晚上这网站就可能会提示网站流量达到上限什么的（然后不管是网页还是我的软件都无法测速了），后来我还发过一个 Issues #6 征集下载测速链接，最后干脆自建了一个用来下载测速的 URL 了。。。

**Comment by XIU2** [comment]
所以你的最终结论是：
> `speed.cloudflare.com` 域名解析的两个固定 IP `104.16.148.27、104.16.149.27` 比自选的更快更稳定么？

我觉得这个结论可能并不适用于所有人（比如我）。。。
